src/cmd/commit.rs

```diff
@@ -294,7 +365,26 @@ impl Command for CommitCommand {
         }
         .wizard(&config, parser, self.interactive)?;
 
-        self.commit(msg)?;
+        std::fs::write(commit_editmsg_path, &msg)?;
+        if !is_git_editor {
+            self.commit_msg_and_remove_file(&msg, commit_editmsg_path)?;
+        }
+
         Ok(())
     }
 }
+
+fn config_types_to_conventional(types: &[Type]) -> Vec<crate::conventional::Type> {
+    types
+        .iter()
+        .map(|ty| ty.r#type.as_str())
+        .map(crate::conventional::Type::from)
+        .collect()
+}
+
+fn get_default_commit_msg_path() -> PathBuf {
+    git2::Repository::open_from_env()
+        .unwrap()
+        .path()
+        .join(".convco_msg")
```

Comment:
Further could be whether or not to make this allcaps like `COMMIT_EDITMSG`, and/or changing it to `convco_editmsg`. But that's definetly a nitpick.